core/geometry/src/polyface/Polyface.ts

```diff
@@ -139,6 +139,55 @@ export class IndexedPolyface extends Polyface { // more info can be found at geo
     else
       this._facetToFaceData = [];
   }
+
+  /**
+   * Given an array of strictly increasing numbers, find the index of the largest number that is less than or equal
```

Comment:
```suggestion
   * Given an array of monotone numbers (i.e., strictly increasing numbers), find the index of the largest number that is less than or equal
```

Reply:
Hmm, monotone means increasing or decreasing. I'll rename in next commit.